### ISSUE TITLE
Controller operation and sound trouble

### ISSUE THREAD

**Issue by yusya1029** [Ryochan7/DS4Windows#572]
In DS4Windows Version1.6.9, a problem occurred.
One is that the button does not respond or it is set differently from the profile on the game despite assigning various buttons in the profile setting. For example, even if I set ○ button of PS4 to be the A button of Xbox, it is changed to × button of PS4 without permission on the game.
Secondly, the sound of the earphones etc. connected to the controller can not be heard at all. With BIOHAZARD RE:2, no sound will be heard as soon as you move through the room.
Please resolve these two problems.

**Comment by Ryochan7** [comment]
Version 1.6.10 should take care of the remapping problem. The second issue would be a bit tricky to diagnose. Which controller are you using or are you using the Sony Wireless Adapter? Headphone jack support is only available if using the Sony Wireless Adapter or using the DS4 v.2 (CUH-ZCT2U) connected via USB. If sound gets disabled at some point then it would seem that DS4Windows cannot grab the current volume from the sound device associated with the headphone jack. The volume will end up getting set to zero when a new output packet is sent to the controller.

**Comment by Ryochan7** [comment]
I do not have the RE2 Remake and my current PC would not be able to run it. I changed the privacy settings for my Steam and GOG profiles so people can now see what is in my libraries. That way people can have a better idea about what I can test against. I should put this info in the README somewhere as well.

https://steamcommunity.com/id/Ryochan7/games/?tab=all
https://www.gog.com/u/Ryochan7/games

**Comment by yusya1029** [comment]
The controller uses PS4 dual shock (CUH-ZCT2U) with USB connection. The thing connected to the headphone jack is Sony made earphone. In version 1.6.7, sound was heard without any problems.
When I tried version 1.6.10 which was released the other day, the above two problems did not occur. The operation of the controller was able to operate according to the profile setting, and the sound was heard firmly without interruption.
I deeply appreciate you.
